ProcessHacker/proctree.c

```diff
@@ -4068,6 +4068,30 @@ BOOLEAN NTAPI PhpProcessTreeNewCallback(
             }
         }
         return FALSE;
+
+    case TreeNewMiddleClick:
+        // selecting subtree works only in NoSortOrder
+        if (ProcessTreeListSortOrder != NoSortOrder)
+            break;
+
+        PPH_TREENEW_MOUSE_EVENT mouseEvent = Parameter1;
+        if (!mouseEvent)
+            break;
+
+        node = (PPH_PROCESS_NODE)mouseEvent->Node;
+
+        // init last index to self, this way we select only the process if there are no children
```

Comment:
please add `(TheEragon)` at the end of these comments so we know who wrote them :P

Reply:
Will do :)